config/admins.txt

```diff
@@ -149,3 +149,4 @@ WaylandSmithy = Game Master
 NamelessFairy = Game Master
 WalterMeldron = Game Master
 san7890 = Game Master
+tigercat2000 = Game Master
```

Comment:
Any reason this was touched?

Reply:
> Any reason this was touched?

Debugging I presume. It has zero effect on live obviously. It’s not really a big deal as long as it’s removed by merge time

Reply:
Completely forgot PR author was tigercat 👍